docs/kubernetes/windows-details.md

```diff
@@ -0,0 +1,335 @@
+# More on Windows and Kubernetes
+
+If you're trying to deploy Kubernetes with Windows the first time, be sure to check out the [quick start](windows.md) first. If you're looking for more details on deployments, examples or troubleshooting read on.

```

Comment:
I would end this sentence using an em-dash:
`...examples or troubleshooting--read on.`

Reply:
Thanks! fixed in new changelist